hax-lib-macros/src/lib.rs

```diff
@@ -474,3 +474,39 @@ pub fn attributes(_attr: pm::TokenStream, item: pm::TokenStream) -> pm::TokenStr
 
     quote! { #item }.into()
 }
+
+/// A marker indicating a `fn` as a ProVerif process read.
+#[proc_macro_error]
```

Comment:
This looks like a lot of boiler plate. If this is intenden, let's throw it into a macro.

Reply:
You're right, it would perhaps be nice to have a `hax_marker!(syn_structure, marker_name)` macro to simplify generation of simple marker attributes for a given token type. My question would be how to do that nicely given that we would still have to manually modify the `AttrPayload` and `AssociationRole` (?) types.

Reply:
Let's leave this for a future PR.

Adding the values to the enums still needs to happen separately unless we want to add more logic around it.